src/Spec2-Core/SpMenuPresenter.class.st

```diff
@@ -28,7 +28,7 @@ Class {
 	#superclass : #SpAbstractWidgetPresenter,
 	#instVars : [
 		'#groups',
-		'#title',
+		'#title => SpObservableSlot',
```

Comment:
what is this title for? (I know you just made it observable, but for me this was in "marked for removal")

Reply:
Titles for menus do not make much sense but because it is there and I wanted to support translation of it, I needed to change it to the observable slot to allow changes propagation